ami/trainers/i_jepa_trainer.py

```diff
@@ -141,20 +142,31 @@ def train(self) -> None:
                     latent_from_target_encoder,
                     reduction="mean",
                 )
+                self.logger.log("i-jepa/target-encoder-latent-std", latent_from_predictor.std(0).mean())
                 self.logger.log("i-jepa/loss", loss)
```

Comment:
`target-encoder-latent-std`の値は学習に必要なlossではなく、あくまで学習がうまくいってるか見るための指標でしかないってことをどこかに明記したい気がする
もしやるならこんな風になるのかな
```suggestion
                self.logger.log("i-jepa/metrics/target-encoder-latent-std", latent_from_predictor.std(0).mean())
                self.logger.log("i-jepa/loss/smooth_l1_loss", loss)
```

Reply:
lossには明示的に "loss"という単語が入るので良いかと。 それにTensorBoardに記録する内容は損失だけじゃないし。
ちなみにTensorBoardは2階層までしかできなかった気がする。まあ今回は別に変えなくてもいいんじゃないかな。

Reply:
今後複数のlossやmetricsを持つモデルが出てきた時、実装してる当人が「lossには明確に"loss"と記述しよう」と必ず思えるかと言われると微妙な気がする
自分的にこの2つは明確に分けるためのルール欲しい
(TensorBoardが2階層までしかできないのは普通に知らなかったごめん)

Reply:
@zassou65535 prefixについてるとやっぱり便利だからつけることにした。2階層までしか階層表示されないけどタグ名として便利なのでそうする。
https://github.com/MLShukai/ami-vconf24/pull/63/commits/79adf89368139719255c1dd59a96e51c1d96284c